src/vaadin-grid-pro-inline-editing-mixin.html

```diff
@@ -43,6 +43,10 @@
         singleCellEdit: {
           type: Boolean,
           notify: true // FIXME(yuriy-fix): needed by Flow counterpart
+        },
+
+        _disableEditing: {
```

Comment:
"disableEditing" sounds like a function whereas this is a property. The naming should be aligned between properties; similarly to "disabled", this should be "_editingDisabled"

Reply:
Done.